db/migrate/1730785487527_create_oauth_access_tokens.rb

```diff
@@ -0,0 +1,20 @@
+class CreateOauthAccessTokens < ActiveRecord::Migration[6.0]
+  def change
+    create_table :oauth_access_tokens do |t|
+      t.references :resource_owner_id, null: false
+      t.string :token, null: false, index: { unique: true }
+      t.string :refresh_token, null: false, index: { unique: true }
```

Comment:
Consider adding an index to the `expires_in` and `refresh_expires_in` columns to improve query performance, especially if these fields are frequently used in queries.